### PR TITLE
[fetcheus] use world as default for fetch :state :worldcoords

### PR DESCRIPTION
**Pull request by knorth55**
PR2 returns relative coords from `/world` in `(send *ri* :state :worldcoords)`,
but Fetch returns relative coords from `/map` in `(send *ri* :state :worldcoords)`.

I fix this problem in this PR, and now Fetch returns relative coords from `/world`.

PR2:
https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus/pr2-interface.l#L88-L96

Current Fetch:
https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus/robot-interface.l#L1790-L1798

cc. @708yamaguchi @mqcmd196

### CODE REVIEW

**Review by 708yamaguchi — COMMENTED** [review]
I checked several `:state :worldcoords` cases with (i. previous version) and (ii. this PR version).
I think we need to deal with the second and fourth case.

- `(send *ri* :state :worldcoords)` is interpreted as:
  1. (send-super :state :worldcoords)
  2. (send-super :state :worldcoords "world")
- `(send *ri* :state :worldcoords :wait-until-update t)` is interpreted as:
  1. (send-super :state :worldcoords :wait-until-update t)
  2. (send-super :state :worldcoords :wait-until-update) -> ERROR!
- `(send *ri* :state :worldcoords "/map")` is interpreted as:
  1. (send-super :state :worldcoords "/map")
  2. (send-super :state :worldcoords "/map")
- `(send *ri* :state :worldcoords "/map" :wait-until-update t)` is interpreted as:
  1. (send-super :state :worldcoords "/map" :wait-until-update t)
  2. (send-super :state :worldcoords "/map")

**Comment by 708yamaguchi** [comment]
I feel `:state :worldcoords` method is very complicated.
Can you add tests like below to `fetcheus/test/test-fetcheus.l`?
```
(deftest fetch-state-worldcoords
  (send *ri* :state :worldcoords)
  (send *ri* :state :worldcoords :wait-until-update t)
  (send *ri* :state :worldcoords "/map")
  (send *ri* :state :worldcoords "/map" :wait-until-update t)
  )
```

**Comment by knorth55** [comment]
we do not have `:wait-until-update` option for `:state :worldcoords`.
I think we do not need to test it.
https://github.com/jsk-ros-pkg/jsk_pr2eus/blob/master/pr2eus/robot-interface.l#L1790-L1799

**Comment by 708yamaguchi** [comment]
OK, thank you.
I did not think `:wait-until-update t` is not used in `:state :worldcoords`, but now I understand.

> Or another solution is that we can add :wait-until-update option for :state :worldcoords.

I think it is good idea, but it should be added by another pull request.
So I approve this pull request.

**Review by 708yamaguchi — APPROVED** [review]

File: jsk_fetch_robot/fetcheus/fetch-interface.l [hunk 1]
```diff
@@ -37,10 +37,13 @@
      ))
   (:state (&rest args)
    ":state calls with :wait-until-update by default, since Fetch publishes /joint_states from body and gripper at almost same frequency"
```
Comment (k-okada) [inline jsk_fetch_robot/fetcheus/fetch-interface.l:39]: @708yamaguchi @knorth55 could you add more documentation here?
```
(send *ri* :state :worldcoords) returns current coordinates in '/world' freame
(send *ri* :state :potentio-vector) ...
...
...
```
or update https://github.com/jsk-ros-pkg/jsk_robot/tree/master/jsk_fetch_robot#common

Comment (708yamaguchi) [reply jsk_fetch_robot/fetcheus/fetch-interface.l:39]: @k-okada 
Thank you for your review.

@knorth55 
Could you check and merge https://github.com/knorth55/jsk_robot/pull/127 ?

Comment (knorth55) [reply jsk_fetch_robot/fetcheus/fetch-interface.l]: Thanks, I merged the PR.

**Pull request merged**